examples/analysis/point_extraction/interpolation.py

```diff
@@ -0,0 +1,56 @@
+"""
+Regular-grid interpolation
```

Comment:
The title is a bit misleading. I would think that you are talking about interpolating irregular points into a regular grid, but you're doing the opposite.
Maybe "Extraction of raster values with interpolation"?

Reply:
Clarified